packages/next-codemod/transforms/name-default-component.ts

```diff
@@ -1,39 +1,44 @@
 import { basename, extname } from 'path'
 
-const camelCase = (value) => {
-  const val = value.replace(/[-_\s.]+(.)?/g, (_match, chr) =>
+const camelCase = (value: string): string => {
+  const val: string = value.replace(/[-_\s.]+(.)?/g, (_match, chr) =>
```

Comment:
```suggestion
  const val = value.replace(/[-_\s.]+(.)?/g, (_match, chr) =>
```

Reply:
Ok, I take care of in after PRs.
Thank you !